api/src/api/userRouter.test.ts

```diff
@@ -259,9 +259,61 @@ describe("userRouter", () => {
         expect(stubTimeStampBusinessSearch.search).not.toHaveBeenCalled();
       });
 
+      it("does not update businessNameAvailability if it's completedFilingPayment and formationResponse success are true", async () => {
+        const userData = generateUserData({
+          formationData: generateFormationData({
+            completedFilingPayment: true,
+            formationResponse: generateFormationSubmitResponse({
```

Comment:
formationResponse isn't the right thing to check here. That doesn't indicate that they have formed/payed, just that they have advanced to the payment screen. 

I think we can remove all references to formationResponse here & in the code and JUST check if `completedFilingPayment` is true

Reply:
okay sounds good!